armory/scenarios/base.py

```diff
@@ -34,11 +36,25 @@ def evaluate(self, config: dict):
         """
         Evaluate a config for robustness against attack.
         """
+        self.setup(config)
         results = self._evaluate(config)
         if results is None:
             logger.warning(f"{self._evaluate} returned None, not a dict")
         self.save(config, results)
 
+    def setup(self, config: dict):
+        """
+        Performs scenario setup on the evaluating system.
+        """
+        if config["sysconfig"].get("external_github_repo", None):
+            external_repo_dir = paths.docker().external_repo_dir
```

Comment:
This line will break when running in `--no-docker` mode, I think, since it will point to `/armory/tmp/external`.